### DISCUSSION TITLE
Sometimes we need support!

### DISCUSSION THREAD

**Discussion by gAlleb** [savonet/liquidsoap#4170] [General]
Hello friends! Hey everyone!

Some of you might have noticed that Matthias (@Moonbase59) hasn't been around as much lately. He's going through a tough time right now and is facing some challenges in his life. He could really use our support.

Matthias has been a huge part of this community for years, volunteering his time and skills to make things better for everyone. He's always been there to help us out, so now it's our turn to show him some love.

If you're able to, please consider donating to help him out. Every little bit counts! You can donate through: 
> [!IMPORTANT]
> ### https://www.paypal.com/donate/?hosted_button_id=PBPR63362LDEU.
<hr>

He's also shared a recent update about his situation on his GitHub page. 
> [!IMPORTANT]
>### Please visit the link to read his latest entry [https://github.com/Moonbase59/autocue/blob/master/IMPORTANT.md](https://github.com/Moonbase59/autocue/blob/master/IMPORTANT.md)  
It gives a better understanding of what he's facing.
<hr>

Thank you for your generosity and for supporting our neighbor in need. **Thank you! ❤️**

**Comment by Tampa** [comment]
I have a spare house that sits close to a city and hospital. It's a little run down and currently has no internet, but I'd be willing to donate that. No one should be left homeless.

**Comment by gavin2812** [comment]
@gAlleb thanks for raising awareness, as someone who comes here often for help from other members, I value everyones help. It's terrible to hear of his situation and would also encourage anyone who can to make a donation, however much they can. Thanks.
